ci/Makefile

```diff
@@ -86,13 +92,13 @@ restart-proxy:
 	  --dry-run) > proxy.log 2>proxy.err & echo $$! > proxy.pid
 	sleep 2 && kill -0 $$(cat proxy.pid)
 
-# Paradox: can't push locally, but ci needs ci-utils.  Just use the latest.
+# Paradox: can't push locally, but ci needs builder_utils.  Just use the latest.
 run-local: PROXY_IP=$(shell gcloud compute instances list --filter=name=dk-test --format="value(EXTERNAL_IP)")
 run-local: restart-proxy
 	HAIL_GCP_PROJECT=$(PROJECT) \
 	  HAIL_DOMAIN=$(DOMAIN) \
 	  HAIL_IP=$(IP) \
-	  HAIL_CI_UTILS_IMAGE=gcr.io/hail-vdc/ci-utils:f477fdaa98d0e640f1bc2003ccfe1d2a86cf2d7531ff595476ca2412d28fc678 \
+	  HAIL_BUILDER_UTILS_IMAGE=gcr.io/hail-vdc/builder:f477fdaa98d0e640f1bc2003ccfe1d2a86cf2d7531ff595476ca2412d28fc678 \  ## FIXME: need to get this!
```

Comment:
I tried to get the hash for the builder docker image I made, but google was down. I think we can worry about that later

Reply:
prefer an issue to fixme

Reply:
Agree, but in this case, we shouldn't merge it until I figure this out. However, I wanted feedback from ci in the meantime

Reply:
Fixed